### PR TITLE
[NTUSER] Fix Skype and other programs installer windows not able to be moved with mouse by clicking on title bar

### PR DESCRIPTION
**Pull request by Doug-Lyons**
Patch by @I_Kill_Bugs

## Purpose

_Fixes Skype installer window not able to be moved with mouse by clicking on title bar_

JIRA issues: [CORE-19487](https://jira.reactos.org/browse/CORE-19487) and [CORE-19538](https://jira.reactos.org/browse/CORE-19538)

## Proposed changes

_Add condition into msgqueue along with filter testing to reject messages that are "NotForUs"._
_Also, do not set AcceptMessage to FALSE, but just keep msgDblClk and idSysPeek values and continue._

### CODE REVIEW

**Comment by binarymaster** [comment]
> [NTUSER] Skype install window not movable by mouse dragging on title bar

Please sync the PR title with your commit title, it looks better:

> [NTUSER] Add condition into msgqueue along with filter testing to reject messages that are "NotForUs".

Also the commit message mentions only one Jira ticket, but there's two.

**Comment by Doug-Lyons** [comment]
> > [NTUSER] Skype install window not movable by mouse dragging on title bar
> 
> Please sync the PR title with your commit title, it looks better:
> 
> > [NTUSER] Add condition into msgqueue along with filter testing to reject messages that are "NotForUs".
> 
> Also the commit message mentions only one Jira ticket, but there's two.

@binarymaster, Can you tell me how to see the actual "commit message"? I believe that according to the information in this ticket, it should show two (2) issues. Thanks.

**Comment by binarymaster** [comment]
image

and https://github.com/reactos/reactos/pull/6819/commits

File: win32ss/user/ntuser/msgqueue.c [hunk 1]
```diff
@@ -1946,6 +1946,7 @@ co_MsqPeekHardwareMessage(IN PTHREADINFO pti,
    DWORD QS_Flags;
    LONG_PTR ExtraInfo;
    BOOL Ret = FALSE;
+   MSG clk_msg;
```
Comment (HBelusca) [inline win32ss/user/ntuser/msgqueue.c:1949]:
```suggestion
   MSG clk_msg;
   BOOL Ret = FALSE;
```
Grouping the variables that save temporary state together.

**Comment by HBelusca** [comment]
Let's see what the testbots think about the PR...

**Comment by Doug-Lyons** [comment]
Testbot results:

VBox: https://reactos.org/testman/compare.php?ids=95140,95144,95148 LGTM
KVM:  https://reactos.org/testman/compare.php?ids=95142,95145,95149 LGTM

**Comment by Doug-Lyons** [comment]
Thanks, @binarymaster. It should be fixed now.

File: win32ss/user/ntuser/msgqueue.c (hunk 1 above)
Comment (Doug-Lyons) [reply win32ss/user/ntuser/msgqueue.c]: Changed. Thanks.

**Comment by Doug-Lyons** [comment]
> I would still suggest to make priority for this description as the PR/commit title:
> 
> > [NTUSER] Add condition into msgqueue along with filter testing to reject messages that are "NotForUs".
> 
> Because it affects not just Skype. You could describe that next to it.

Changed to show "Skype and other programs" now. Thanks.

**Review by HBelusca — APPROVED** [review]

**Review by cbialorucki — APPROVED** [review]

**Pull request merged**